cbmc/proofs/Makefile.common

```diff
@@ -83,6 +83,7 @@ CBMCFLAGS += \
 	--undefined-shift-check \
 	--signed-overflow-check \
 	--unsigned-overflow-check \
+	--flush \
```

Comment:
`--flush`?

Reply:
As above, it is essential to run with --flush so that output is flushed to standard output as soon as it is available.  We don't want to be stuck wondering where CBMC is, whether CBMC has or has not entered the SAT phase, and the "Running SAT" comment is just sitting in the output buffer.